packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -115,16 +121,37 @@ const BlockActionsMenu = ( {
 		wrapBlockSettings && settingsOption,
 		copyButtonOption,
 		cutButtonOption,
-		isPasteEnabled && pasteButtonOption,
+		isPasteEnabled() && pasteButtonOption,
 		duplicateButtonOption,
 		deleteOption,
 	] );
 
+	function onPasteBlock() {
+		if ( ! clipboard ) {
+			return;
+		}
+
+		pasteBlock( rawHandler( { HTML: clipboard } )[ 0 ] );
+	}
+
+	function isPasteEnabled() {
```

Comment:
It doesn't look like this is used anywhere else. Could we make it a `const`?

Reply:
What do you mean by making it a `const`?

Reply:
I mean it's only called once in the `options` declaration a few lines above, we don't need to make it a function.
```js
const clipboardBlock = rawHandler( { HTML: clipboard } )[ 0 ];
const isPasteEnabled = clipboardBlock && canInsertBlockType( clipboardBlock.name, rootClientId )
```

Reply:
Gotcha, thanks! Updated it.

One thing though, I had to add a local state to the Block Actions menu component to store the current clipboard. For cases like:

Select a block > Open the actions menu > Tap on copy > Open the actions menu of the same block > `Paste after` action should show up.

Reply:
It's a bit confusing but I guess it's a cool workaround :+1:
Let's revisit when we actually use the phone clipboard